### ISSUE TITLE
Anvil GUIs don't work anymore

### ISSUE THREAD

**Issue by klaidoshka** [tr7zw/YAPFA#473]
**Expected behavior**
After using certain command to open AnvilGUI and writing something in it to get the wanted output, everything works well. The output command executes.

**Observed behavior**
After using certain command to open AnvilGUI and writing something in it, no output is being executed, the "protected" items that are within the GUI can be taken out, not like before.

**Steps to reproduce (if known)**
Install any Yatopia version after (including) this:
https://github.com/YatopiaMC/Yatopia/actions/runs/772102895

**Installed plugins**
The MyCommand alone is enough to have for this to happen.

**Version**
Can't do a screenshot, so... This and any more up-to-date:
https://github.com/YatopiaMC/Yatopia/actions/runs/772102895

**In short**
The builds before that one were working fine, after that build everything went nuts, I have no clue if it's something on MyCommand dev's side to do/update or can anything be done from your side 😞

Edit: latest upstream fixed it 👍
